eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -16,6 +16,14 @@
     <UsagePattern IdentityGlob="System.Diagnostics.EventLog/7.0.0" />
     <UsagePattern IdentityGlob="System.Security.Cryptography.ProtectedData/7.0.0" />
 
+    <UsagePattern IdentityGlob="Microsoft.AspNetCore.App.Runtime.linux-x64/8.0.1" />
+    <UsagePattern IdentityGlob="Microsoft.AspNetCore.App.Runtime.win-x64/8.0.1" />
+    <UsagePattern IdentityGlob="Microsoft.NETCore.App.Crossgen2.linux-x64/8.0.1" />
+    <UsagePattern IdentityGlob="Microsoft.NETCore.App.Host.win-x64/8.0.1" />
+    <UsagePattern IdentityGlob="Microsoft.NETCore.App.Runtime.linux-x64/8.0.1" />
+    <UsagePattern IdentityGlob="Microsoft.NETCore.App.Runtime.win-x64/8.0.1" />
```

Comment:
Why are those needed now?

Reply:
Because we now native compile proto and coreclr fsc and fsi rather than just build to IL.

Reply:
Why is anything with `AspNetCore` in this list though?